mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala

```diff
@@ -77,6 +77,22 @@ class KMeansModel @Since("1.1.0") (@Since("1.0.0") val clusterCenters: Array[Vec
   def predict(points: JavaRDD[Vector]): JavaRDD[java.lang.Integer] =
     predict(points.rdd).toJavaRDD().asInstanceOf[JavaRDD[java.lang.Integer]]
 
+  /**  */
+  def distanceToCenters(point: Vector): (Int, Double) = {
```

Comment:
Indeed not. The diff I received must have been a work in progress. This is now fixed.